src/app/store/hooks/usePost.tsx

```diff
@@ -12,8 +14,8 @@ export const usePost = ( id: number ): Post => {
         meta: {
             _thumbnail_id: 2,
         },
-        category: 1,
-        author: 1,
+        categories: [useCategory(1)],
```

Comment:
Spacing here is not correct, make sure spacing is automatically fixed in your editor. It should happen automatically if you have TSLint installed in vscode.